dd-sdk-android-okhttp/src/main/kotlin/com/datadog/android/okhttp/DatadogInterceptor.kt

```diff
@@ -65,21 +65,23 @@ import java.util.Locale
  * ```
  *     val tracedHosts = listOf("example.com", "example.eu")
  *     OkHttpClient client = new OkHttpClient.Builder()
- *         .addInterceptor(new DatadogInterceptor(tracedHosts))
+ *         .addInterceptor(new DatadogInterceptor(sdkCore, tracedHosts))
  *         // Optionally to get information about redirections and retries
- *         // .addNetworkInterceptor(new TracingInterceptor(tracedHosts))
+ *         // .addNetworkInterceptor(new TracingInterceptor(sdkCore, tracedHosts))
  *         .build();
  * ```
  */
 open class DatadogInterceptor
 internal constructor(
+    sdkCore: SdkCore,
```

Comment:
btw we also have a task to use lazy APIs for SDK instance/sampling here, will be done later.